README.md

```diff
@@ -87,26 +87,25 @@ Licensed under the Apache License, Version 2.0. See: [Apache License](http://www
 
 ## About Split
  
-Split is the leading Feature Delivery Platform for engineering teams that want to confidently deploy features as fast as they can develop them. Split’s fine-grained management, real-time monitoring, and data-driven experimentation ensure that new features will improve the customer experience without breaking or degrading performance. Companies like Twilio, Salesforce, GoDaddy and WePay trust Split to power their feature delivery.
+Split, now a part of Harness, is a leading Feature Delivery Platform for engineering teams that want to confidently deploy features as fast as they can develop them. Split’s fine-grained management, real-time monitoring, and data-driven experimentation ensure that new features will improve the customer experience without breaking or degrading performance. Companies like Twilio, Salesforce, GoDaddy and WePay have trusted Split to power their feature delivery.
```

Comment:
Should we just call it "Harness Feature Management and Experimentation (formerly Split)" or something?